packages/workbox-build/src/lib/generate-sw.js

```diff
@@ -18,6 +18,8 @@ const constants = require('./constants');
  * `globPatterns` against.
  * @param {Array<String>} input.globPatterns Files matching against any of
  * these glob patterns will be included in the file manifest.
+ *
+ * Defaults to ['**\/*.{js,css}']
```

Comment:
Okay, gotcha. This was something that ESLint was complaining about when it made it into the actual code rather than the comments, so I was on the lookout for it.